SECURITY.md

```diff
@@ -0,0 +1,58 @@
+# Security Policy
+
+## Supported Versions
+
+Security is a top priority for our project. We are committed to providing updates and addressing security vulnerabilities in a timely manner. Below is a list of versions of our project currently receiving security updates.
+
+| Version | Supported          |
+| ------- | ------------------ |
+| 23.x    | :white_check_mark: |
+| 22.x    | :white_check_mark: |
```

Comment:
@zware agreed on strategy long term of only supporting most recent release. 

My thinking for now was that 22.3.5 was a "major" release if you will where many users probably installed that version over the past two years and I think before "not supporting" that version, which many of our end users have installed, may not be the best experience.

I'd suggest that we announce an end to support of multiple versions more publicly so people can prepare and then we can remove that targeting more frequent and smaller releases. 

What are your thoughts on that strategy?

Reply:
We've never announced a support policy as far as I know (beyond the standard open source "no warranty expressed or implied" :)).  `pycountry` has historically been careful to maintain backwards compatibility such that the only practical difference between even "major" versions is updated data and possibly new features, which I think we would do well to continue.

As far as smaller and more frequent releases, I would urge caution.  We're not a small package; we lost a lot of weight from 22.3.5->23.12.7, but at 6MB per wheel, we can still start adding up -- not nearly as quickly as something like `torch`, but it's still worth considering.  I could get behind the idea of splitting the monolithic `pycountry` package into "`pycountry-code`" (probably still just named `pycountry`) and `pycountry-data` as an independently-updated dependency that just holds the data.  That would allow us to update the code more frequently without duplicating the data every time, and would also make it more feasible to support multiple versions of the code.